### DISCUSSION TITLE
YAMAHA 最新 AI 鋼琴家：與真人共同演奏，人類彈錯他還能迅速 cover

### DISCUSSION THREAD

**Discussion by 32844583** [1101-GS3073-C/Python-Community-GS3073-C#522] [General]
### YAMAHA「AI 合奏系統」實現與已逝偉大音樂家合奏的夢想
人工智能合奏系統是一種利用人工智能技術來協助人類音樂家進行合奏的系統。這種系統通過分析人類演奏者的動作，及時預測出最合適的伴奏方式，從而幫助人類音樂家實現最佳的合奏效果。這種系統的優勢在於，它可以適應人類演奏者的技術水平，即使人類演奏者技術較差，AI也能夠配合彈得較差。此外，人工智能合奏系統還能夠應用於各種樂器，比如鋼琴和管弦四重奏。這使得人工智能合奏系統成為了一種非常有效的工具，可以幫助人類音樂家實現最佳的合奏效果。
https://buzzorange.com/techorange/2017/12/18/yamaha-ai-pianoist/

**Comment by NcuJerryKao** [comment]
AI 電腦運轉的速度遠比人類快 面對臨場危機 還是靠AI更可靠
有人彈錯可馬上Cover真是太強了

**Comment by williamyen0623** [comment]
哇！真的好酷！說不定之後還能跟那些已逝的巨星歌手同台演唱，想到就興奮！

**Comment by BingRong1217** [comment]
AI結合音樂可以帶給我們更好的享受

**Comment by Ben6uo** [comment]
感覺這東西太強了 有AI幫助一定超好聽

**Comment by awehao** [comment]
人類的藝術細胞意義何在，各個叫苦連天，集體罷工，既然我們音樂，繪畫，各種東西都有了AI，人類以後會不會演化成笨蛋啊

**Comment by bww0401** [comment]
雖然ai可以乍看之下可以表演的比人好，但是我認為藝術跟音樂都是情感的體現，而人工智慧是沒有辦法傳達的，這也是人類無法完全被替代的主要原因

**Comment by manipulateed** [comment]
這樣會不會讓人再有合奏的情況，忽視自己還不夠好的地方啊?但這個功能也確實很人性化，還可以根據使用者的能力去配合調整。

**Comment by larry40305** [comment]
印象中資工系也有在研究如何利用AI創作音樂，但我都沒在上課所以忘記了。

**Comment by awehao** [comment]
人類的藝術細胞意義何在，各個叫苦連天，集體罷工，既然我們音樂，繪畫，各種東西都有了AI，人類以後會不會演化成笨蛋啊

**Comment by awkward-willy** [comment]
我雖認同前面所說音樂等藝術創作之所以美妙是因為背後深刻的情感，但這些情感往往是透過音樂的強弱，停頓等來表現，或許在機器學習不斷訓練，以及科技不斷演進下，所謂藝術真的會被各式各樣的技術所取代。

**Reply by RogelioKG** [reply]
我是認為 AI 之所以能做出一般大眾都喜歡的作品，
是因為我們丟進去的訓練集，都是這樣的取向。
這也是為什麼 AI 很難產生獨特的風格，
因為獨特作品的訓練集很少，
這就產生了選擇偏差 (Selection bias)

**Reply by RogelioKG** [reply]
所以說 AI 比較難產生的，應該是風格的創新，
而藝術、美術、音樂總是有不斷的風格創新湧現。
不過若只是要產出現有風格的作品，
AI 應該仍然是不錯的載體。
